auth/token.go

```diff
@@ -15,6 +15,8 @@ import (
 )
 
 const (
+	claimCnf             = "cnf"
+	claimCnfMemberX5t    = "x5t#S256"
```

Comment:
I simplified the constant slighlty in java -> CNF_X5T

Reply:
thanks for the hint 👍